src/pages/AuthenticationPage/authFunctions.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from "react";
+
+
+const useInputHandler = (state) => {
+  const [inputState, setInputState] = useState(state);
+  const inputUpdate = (e) => {
+    const inpValue = e.target.value;
+    setInputState({
+      ...inputState,
+      [e.target.name]: inpValue,
+    });
+  };
+  return { inputState, inputUpdate };
+};
```

Comment:
appreciation - I liked how you have separated the auth functions

Reply:
Thanks